operations/boot_orchestration/Rolling_Upgrades.md

```diff
@@ -25,6 +25,8 @@ For more information on staging sessions, see [Creating a Staged Session](./Stag
 1. The workload manager calls the `applystaged` endpoint for each node when it is ready.
 BOS then copies the information staged for these components into their desired state, and BOS starts to operate on the nodes and attempts to make their actual state match their new desired state.
 
+<a name="staged-sessions-with-slurm"></a>
```

Comment:
I don't believe you need this. You should just be able to reference this section with something like the following from `operations/iuf/workflows/managed_rollout.md`:

```
[Using staged sessions with Slurm](../../operations/boot_orchestraction/Rolling_Upgrades.md#using-staged-sessions-with-slurm)
```

Reply:
Changed.